tests/selenium/cypress/integration/header_spec.js

```diff
@@ -0,0 +1,22 @@
+import MainPage from '../page-objects/MainPage';
+
+describe('header sanity check', () => {
```

Comment:
Do these specs replace some old protractor tests? If so, can you remove the matching protractor tests to show that?

Reply:
done